todo_app/app.py

```diff
@@ -1,11 +1,54 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
+from .data.trello_items import *
 from todo_app.flask_config import Config
 
+import os
+
+
+from flask_wtf import FlaskForm
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+boardID = "645106104aeafb96fe026623"
+todoID = "6451065f9818f6f58c290efb"
+doneID = "64f1e683328cb18730751208"
```

Comment:
While these aren't sensitive, they are likely to vary between different environments (e.g. we'll use a different board to back development than production!) so loading these in from environment variables (like the secrets discussed below) will be important